tools/raku-on-windows/raku-on-windows.pod6

```diff
@@ -0,0 +1,65 @@
+=begin pod
+
+=head1 Raku / Perl 6 Installation Guide for Windows
+
+Z<2021-11-06, initial text contributed by Todd>
+
+=head2 Step 1. Remove prior installation(s)
+
+If there exists a prior installation of Perl 6 / Raku on your
+computer, remove it.  Current installers are not adept at removing
+prior installations so it is recommended to remove them manually.
+
+Go into the B<Control Panel, Programs and Features>, and remove prior installations
+
+=begin code
+<ctrl><R> AppWiz.cpl
+=end code
+
+=head2 Step 2. Install dependencies
+
+Windows installers do not come with a "spec file" to automatically
+download and install dependencies as does Linux packages, such as
+'dnf/rpm'.  If not included in the installer, it is up to the user to
+figure them out on his own.  Perl 6 / Raku comes with such a dependency
+for 'git', which is required to comfortably install packages with
+the Raku module installer 'zef'.
+
+To install "git", go to
+
+ L<https://gitforwindows.org>
+
+and scroll down from the top.
+
+Note: to add git's UNIX (Linux) tools (ls, tar, curl, etc.)
+      to the path.  Accept all the other defaults.
+
+1. inside the installer, on the
+   + Adjust your PATH environment (window name at top)
+         - Use Git and optional Unix tools from the Command
+         Prompt (bottom selection)
+
+P<file:./tool/raku-on-windows/pics/Raku-WinInstall-fig-1-git-setup.png>
+
+=begin code
+2) Or, after the fact:
+   --> <win><r> sysdm.cpl
+     --> Advanced (tab at the top)
+      --> Environmental Variables (button, lower right)
+       --> Path, Edit, add to the end
+                ;C:\Program Files\Git\usr\bin
+=end code
+
+Note: any command terminals opened before the installation will have to be reopened to update the path to the Git commands.
+
+
+## Install Perl 6 / Raku:
+
+Install Raku (Perl 6) for Windows:
+Note: only the 64 bit version is supported.  For a 32 bit version, as the mailing list for help.
```

Comment:
"ask" the mailing list. Also, I'd remove this note and say that 64 bit is the supported version; if we have instructions, let's add them; if not, I don't think the mailing list is going to help.

Reply:
Feel free to edit as necessary. This first chunk was a rough translation from Todd's input and I haven't had a chance to do anything else with it.